### PR TITLE
Create datebuilder and tests for BlockDate and FreeDate

### PR DESCRIPTION
**Pull request by yewwoei**

### CODE REVIEW

File: src/test/java/systemtests/accounting/AddDebtCommandSystemTest.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import seedu.address.testutil.UserBuilder;
 import systemtests.AddressBookSystemTest;
 
+
```
Comment (evanmok2401) [inline src/test/java/systemtests/accounting/AddDebtCommandSystemTest.java]: Could you remove the extra line?

Comment (evanmok2401) [reply src/test/java/systemtests/accounting/AddDebtCommandSystemTest.java]: Good Job btw. Coverage is now 63% woohoo

Comment (yewwoei) [reply src/test/java/systemtests/accounting/AddDebtCommandSystemTest.java]: approve first, I'll fix this in the next PR

Comment (yewwoei) [reply src/test/java/systemtests/accounting/AddDebtCommandSystemTest.java]: That's it?

Comment (evanmok2401) [reply src/test/java/systemtests/accounting/AddDebtCommandSystemTest.java]: Okay! Just fix it in the next!

**Pull request merged**